### DISCUSSION TITLE
2023-08-07

### DISCUSSION THREAD

**Discussion by woosung1223** [woowacourse-teams/2023-haru-study#268] [daily_meeting]
### 오늘의 할 일
- ~~버그 리포팅 1:30 ~ 2:30 (공통)~~
- ~~로그인 기능 관련 기획 회의 (2시간 정도)~~
- ~~스터디 참여 및 개설 관련 API 회의 (엽토랑)~~

### 언젠가 할 일
- 소켓, 폴링 양방향 통신 어떻게 할지
- 기획 변경된 이유 정리
- 질문 PlaceHolder로 유의미한 응답 예시를 제공하는 기능 (목표설정 가이드라인)
- 브랜치 전략 결정하기(dev -> main)
- A라는 스터디를 참여후 B스터디 참여한 뒤 다시 A스터디에 참여할 시 닉네임을 찾지 못한 이슈(로그인이 된다면 해결될 문제이긴 함)

### 개인 일정
- 근로 3:00 ~ 4:00 (노아, 마코)

**Comment by woosung1223** [comment]
### 체크인
8점 - 월요일인데 엄청 더워서 -2점입니다 🥲

### 체크아웃
7점 - Oauth 플로우가 생각보다 많이 복잡해서 추가로 -1점 감점입니다..

**Comment by jaehee329** [comment]
### 체크인
8점 - 무난한 월요일인데 시간이 별로 없네요 우리

### 체크아웃
5점 - 회의가 잘 안되네요

**Comment by nlom0218** [comment]
### 체크인

9점 - 진짜 웬만하면 덥다고 뭐라하지 않는데, 너무 더워서 한 마디 해야겠어요. 너무 더워요

### 체크아웃

10점 - 역시 회의하니까 살아있다라는 것을 느껴요

**Comment by MoonJeWoong** [comment]
### 체크인
10점 - 어제 잠을 많이 못자고 왔는데 방금 30분 정도 기절했더니 개운해져서 기분이 좋네요!

### 체크아웃
9점 - 로그인이 생각보다 할게 굉장히 많다는 생각이 들어서 일정 추정에 관한 걱정이 조금 생겨납니다... 😢

**Comment by aak2075** [comment]
### 체크인

10점 - 즐거운 월요일입니다💪

### 체크아웃

10점 - 회원이냐 비회원이냐 그것이 문제로다

**Comment by yeopto** [comment]
### 체크인

10점 - 푹 쉬다 와서 좋습니다.

### 체크아웃

6점 - 로그인 기능 회의도 힘드네요..
